packages/server/graphql/mutations/updateCommentContent.ts

```diff
@@ -47,8 +48,8 @@ export default {
       return {error: {message: `Not a member of the meeting`}}
     }
     const {createdBy} = comment
-    if (createdBy !== viewerId) {
-      return {error: {message: 'Can only update your own comment'}}
+    if (createdBy !== viewerId && createdBy !== PARABOL_AI_USER_ID) {
```

Comment:
Theoretically, I can provide my own meeting id, but `commentId` from any other meeting of any other user. And if it is AI comment, I can edit it.
Can we check that the comment belong to the meeting?